src/libraries/System.Numerics.Tensors/tests/TensorSpanTests.cs

```diff
@@ -1704,6 +1704,9 @@ public static void TensorSpanTryCopyTest()
         [Fact]
         public static void TensorSpanSliceTest()
         {
+            // Make sure slicing an empty TensorSpan works
+            new TensorSpan<double>(Array.Empty<double>()).Slice(new NRange[] { .. });
```

Comment:
The test should validate the shape of the resulting span. There should also be variations for empty spans of differing shapes (not just 0 dimensions).

Reply:
Fixed.